src/lib.rs

```diff
@@ -62,6 +75,36 @@ pub fn ansi(tokens: TokenStream) -> TokenStream {
                         "magenta" => "35",
                         "cyan" => "36",
                         "white" => "37",
+                        "brightblack" => "90",
+                        "gray" => "90",
+                        "grey" => "90",
+                        "brightred" => "91",
+                        "brightgreen" => "92",
+                        "brightyellow" => "93",
+                        "brightblue" => "94",
+                        "brightmagenta" => "95",
+                        "brightcyan" => "96",
+                        "brightwhite" => "97",
+                        // background colors
+                        "on_black" => "40",
+                        "on_red" => "41",
+                        "on_green" => "42",
+                        "on_yellow" => "43",
+                        "on_blue" => "44",
+                        "on_magenta" => "45",
+                        "on_cyan" => "46",
+                        "on_white" => "47",
+                        "on_brightblack" => "100",
+                        "on_gray" => "100",
+                        "on_grey" => "100",
+                        "on_brightred" => "101",
+                        "on_brightgreen" => "102",
+                        "on_brightyellow" => "103",
+                        "on_brightblue" => "104",
+                        "on_brightmagenta" => "105",
+                        "on_brightcyan" => "106",
+                        "on_brightwhite" => "107",
```

Comment:
What do you think about making these options more "human"? That is, for example, instead of the option being `on_brightred`, it could be `on bright red`? This should take only a slight adjustment to the regex to accept spaces (`r"\ "`).

Reply:
Yes, that is fine too. I'll change that.